src/env.h

```diff
@@ -45,6 +45,7 @@
 #include <cstdint>
 #include <functional>
 #include <list>
+#include <stack>
```

Comment:
I think this can be removed since an Array is used now

Reply:
Yep, fixed.